models/recordsModel.js

```diff
@@ -0,0 +1,38 @@
+import mongoose from "mongoose";
+const recordSchema = new mongoose.Schema(
+  {
+    questionId: { type: String, required: true, unique: false },
+    answers: [{ type: Number, required: true }]
+  },
+  { timestamps: true }
+);
+
+const RecordModel = mongoose.model("Record", recordSchema);
+
+const save = async model => new RecordModel(model).save();
+const getAllRecords = async () => RecordModel.find();
+const getRecordById = async _id => RecordModel.findById(_id);
+
+const updateById = async (recordId, update) =>
+  RecordModel.findByIdAndUpdate(recordId, update, { new: true });
+
+const deleteRecordById = async recordId =>
+  RecordModel.findByIdAndDelete(recordId);
+
+const getRecordsFromIdArray = async _id => {
```

Comment:
name of the parameter should be meaningful. idArray.
you need to map idArray to be an array of mongoose.Types.ObjectId with id in it
then you need to pass that array into find function with flag $in.